src/emulator/modules/SceDisplay/SceDisplay.cpp

```diff
@@ -77,53 +90,43 @@ EXPORT(int, sceDisplayUnregisterVblankStartCallback) {
 EXPORT(int, sceDisplayWaitSetFrameBuf) {
     STUBBED("move after setframebuf");
 
-    if (!host.display.sync_rendering) {
-        std::unique_lock<std::mutex> lock(host.display.mutex);
-        host.display.condvar.wait(lock);
-    } else
-        host.display.condvar.notify_all();
-
-    if (host.display.abort.load())
-        return SCE_DISPLAY_ERROR_NO_PIXEL_DATA;
-
-    return SCE_DISPLAY_ERROR_OK;
+    return display_wait(host);
 }
 
 EXPORT(int, sceDisplayWaitSetFrameBufCB) {
-    return UNIMPLEMENTED();
+    STUBBED("move after setframebuf");
+
+    return display_wait(host);
 }
 
 EXPORT(int, sceDisplayWaitSetFrameBufMulti) {
-    return UNIMPLEMENTED();
+    STUBBED("move after setframebuf");
+
```

Comment:
weird blank line

run clang-format pls, if not remove the line

Reply:
i have run it, and change nothing, because it normal, is just space for separe information

Reply:
There is no information in the blank line. If we are gonna add more, we gonna do it later. Remove this unnecessary blank

Reply:
is not unnecessary , is just for separe Stub message and return value, never fusion both, need space, is ugly without :/

Reply:
You said it at one place. Stop babbling here

Reply:
oups yes sorry